slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/InsertTest.scala

```diff
@@ -182,7 +183,7 @@ class InsertTest extends AsyncTest[JdbcTestDB] {
     DBIO.seq()
   }
 
-  def testInsertOrUpdatePlainWithFuncDefinedPK: DBIOAction[Unit, _, _] = {
+  def testInsertOrUpdatePlainWithFuncDefinedPK: DBIOAction[Unit, NoStream, Effect.All] = {
```

Comment:
FYI, I have reported https://github.com/scala/bug/issues/11491 as a result of the investigation on why this did not compile in scala 2.13.